subsys/logging/log_output_syst.c

```diff
@@ -1180,6 +1212,30 @@ void log_output_msg2_syst_process(const struct log_output *output,
 
 	update_systh_platform_data(&log_syst_handle, output, flag);
 
+#ifdef CONFIG_LOG_MIPI_SYST_OUTPUT_LOG_MSG_SRC_ID
+	uint8_t level = log_msg2_get_level(msg);
+	bool raw_string = (level == LOG_LEVEL_INTERNAL_RAW_STRING);
+	int16_t source_id = -1;
```

Comment:
maybe initialize it to CONFIG_LOG_MIPI_SYST_MSG_DEFAULT_MODULE_ID?

Reply:
Updated.